### DISCUSSION TITLE
ArgoCD Terminating the container when should sync

### DISCUSSION THREAD

**Discussion by AleixoLucas42** [argoproj/argo-cd#8816] [General]
Hello Everyone,

I have a CICD pipeline that have some CI / tests and inside my pipeline i've a step thats change the deployment yaml file and do a git commit to the repository where my argoCD app is listening. After the pipeline change my deployment file, I send a sync through API to sync the ArgCD App, but when start to create a new container, the container sudenly terminate and the version keep the last version. This started to occur when I updated to version 2.0.5 +

Any Ideas that I can fix this?

My ArgoCD APP Manifest:

`project: ecommerce
source:
  repoURL: 'my repo using SSH'
  path: .
  targetRevision: master
destination:
  server: 'My Kubernetes:443'
  namespace: My Namespace
syncPolicy:
  automated: {}
  retry:
    limit: 10
    backoff:
      duration: 10s
      factor: 2
      maxDuration: 4m0s
`

**Comment by kostis-codefresh** [comment]
You need to provide some more details for this issue.

 
- What is the Application YAML?
- What is the deployment YAML?
- What do you see after the commit?
- What are the logs for Argo CD?
- What are the logs of the application?
